.github/workflows/additional.yml

```diff
@@ -18,9 +18,9 @@ jobs:
       - name: Setup Conda Environment
         uses: conda-incubator/setup-miniconda@v3.0.4
         with:
-          miniforge-variant: Mambaforge
           miniforge-version: latest
           use-mamba: true
+          mamba-version: "1.5.10"
```

Comment:
We can undo this pin once https://github.com/conda-incubator/setup-miniconda/issues/366 is resolved

Reply:
Actually, it turns out this pin wasn't needed. We just needed to switch to miniforge. I've removed these pins.